File: read_excel.js
const xlsx = require('xlsx');
try {
  const wb = xlsx.readFile('./managers emails.xlsx');
  const sheet = wb.Sheets[wb.SheetNames[0]];
  const data = xlsx.utils.sheet_to_json(sheet);
  console.log(JSON.stringify(data.slice(0, 10), null, 2));
} catch (e) {
  console.error("Error:", e.message);
}

File: read_excel.py
import pandas as pd
import json
import sys

try:
    df = pd.read_excel('managers emails.xlsx')
    data = df.head(10).to_dict('records')
    with open('temp_output.json', 'w', encoding='utf-8') as f:
        json.dump(data, f, ensure_ascii=False, indent=2)
    print("Success")
except Exception as e:
    with open('temp_output.json', 'w', encoding='utf-8') as f:
        f.write("Error: " + str(e))
    print("Error")
